x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx

```diff
@@ -169,24 +170,25 @@ describe('policy table', () => {
     expect(buttons.at(2).text()).toBe('Delete policy');
     expect((buttons.at(2).getDOMNode() as HTMLButtonElement).disabled).toBeTruthy();
   });
-  test('should have proper actions in context menu when there are not linked indices', () => {
+  test('view indices button should be disabled when there are no linked indices', () => {
     const rendered = openContextMenu(1);
     const buttons = rendered.find('button.euiContextMenuItem');
-    expect(buttons.length).toBe(2);
-    expect(buttons.at(0).text()).toBe('Add policy to index template');
-    expect(buttons.at(1).text()).toBe('Delete policy');
+    expect(buttons.length).toBe(3);
+    expect(buttons.at(0).text()).toBe('View indices linked to policy');
+    expect(buttons.at(1).text()).toBe('Add policy to index template');
+    expect(buttons.at(2).text()).toBe('Delete policy');
     expect((buttons.at(1).getDOMNode() as HTMLButtonElement).disabled).toBeFalsy();
```

Comment:
Same question here - is this testing the correct thing? It looks like the view indices button is at index `0` and disabled should be `true`.

```
    expect((buttons.at(0).getDOMNode() as HTMLButtonElement).disabled).toBeTruthy();
```

Reply:
I updated the button tests, but also plan on working more on test refactoring in a future PR.